src/test/java/chess/RookTest.java

```diff
@@ -23,9 +26,80 @@ public void testBlackRookToString() {
     @Test
     public void testWhiteRookToString() {
         Position pos = new Position(0, 0);
-        Rook piece = new Rook(Color.WHITE, pos);
+        ChessBoard chessBoard = new ChessBoard();
+        Rook piece = new Rook(Color.WHITE, pos, chessBoard);
         String expected = "RW";
         String actual = piece.toString();
         assertEquals(expected, actual);
     }
+    @Test
+    public void testMoveUpRook() {
+        ChessBoard chessBoard1 = new ChessBoard();
+        chessBoard1.crateAnEmptyChessBoard();
+        ChessBoard chessBoard = new ChessBoard(new Rook(Color.WHITE,new Position(4,4),chessBoard1));
+        Rook piece = (Rook) chessBoard.getBoard()[4][4];
+        boolean po= piece.move(new Position(7,4));
+        assertTrue(po);
+    }
+    @Test
+    public void testMoveDownRook() {
+        ChessBoard chessBoard1 = new ChessBoard();
+        chessBoard1.crateAnEmptyChessBoard();
+        ChessBoard chessBoard = new ChessBoard(new Rook(Color.WHITE,new Position(4,4),chessBoard1));
+        Rook piece = (Rook) chessBoard.getBoard()[4][4];
+        boolean po= piece.move(new Position(0,4));
```

Comment:
dont forget to use comas
and this implementation is a little strange, try to use just one chessboard

Reply:
i made the changes could you review it please?